src/client-api/ActionService/index.js

```diff
@@ -0,0 +1,6 @@
+const BaseService = require("../BaseService");
+
+class ActionService extends BaseService {
```

Comment:
Why not use base service directly since this one has nothing different than its super class?

Reply:
BaseService shouldn't be used as a class of it's own, it only exists to extend. These are empty right now because we will start to add content when we need to use them for anything more complex than getting all/the id, so we'll end up creating these files anyway.